website/static/js/pointers.js

```diff
@@ -69,9 +75,13 @@ var AddPointerViewModel = oop.extend(Paginator, {
                 self.currentPage(result.page);
                 self.numberOfPages(result.pages);
                 self.addNewPaginators();
+                self.loadingResults(false);
```

Comment:
This should probably go in the `always` callback; we want to hide the loading indicator whether or not we get a success or a failure response.